### PR TITLE
docs: Self-hosted kubernetes installation page refactor

### PR DESCRIPTION
**Pull request by ppawlowski**
## Description

This pull request refactors documentation related to FlowFuse installation on Kubernetes environment.
New layout is aligned with a one applied on a Docker installation page. Also, it solves problems defined during evaluation of current installation description.

## Related Issue(s)

Closes https://github.com/FlowFuse/flowfuse/issues/4634

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] Suitable unit/system level tests have been added and they pass 
 - [ ] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: docs/install/kubernetes/README.md [hunk 1]
```diff
@@ -11,92 +11,99 @@ meta:
...
 
 ### DNS
 
 A [wildcard DNS entry](https://en.wikipedia.org/wiki/Wildcard_DNS_record) will be needed 
 to point to the domain that is used for the project instances. This will need to point 
-to the K8s Ingress controller.
+to the kubernetes Ingress controller.
 
-For example if you want projects to be access able as `[project-name].example.com`
+For example if you want projects to be accessible as `[project-name].example.com`
```
Comment (hardillb) [inline docs/install/kubernetes/README.md:106]:
```suggestion
For example if you want projects to be accessible as `[instance-name].example.com`
```

File: docs/install/kubernetes/README.md [hunk 2]
```diff
@@ -105,132 +112,241 @@ that you set the domain to `example.com`.
...
+Please refer to the [Flowfuse Helm Chart documentation](https://github.com/FlowFuse/helm/blob/main/helm/flowforge/README.md#upgrading-chart) for more details
+about the upgrade process.
+
+## Common Questions
+
+### I would like to secure the platform with HTTPS, how can I do that?
+
+In cloud environments, it is recommended to use a Load Balancer to terminate SSL traffic.
+
+However, if you want to use SSL termination on the Kubernetes Ingress Controller, we give such possibility by utilizing [Cert-Manager](https://cert-manager.io/docs/) tool (not part of the FlowFuse Helm chart).
```
Comment (hardillb) [inline docs/install/kubernetes/README.md:199]:
```suggestion
However, if you want to use SSL termination on the Kubernetes Ingress Controller, this is possible by utilizing [Cert-Manager](https://cert-manager.io/docs/) tool (not part of the FlowFuse Helm chart).
```

**Review by hardillb — COMMENTED** [review]
Added some comments.

I'm still looking, but it's hard to render the check list locally in VSCode at the moment

**Comment by joepavitt** [comment]
> I'm still looking, but it's hard to render the check list locally in VSCode at the moment

If you run the website locally, you'll see it all rendered live.

File: docs/install/kubernetes/README.md [hunk 3]
```diff
@@ -105,132 +112,241 @@ that you set the domain to `example.com`.
...
-  email:
-    ses:
-      region: eu-west-1
-```
-
-A more detailed example for running on AWS can be found [here](aws.md)
+### Label Nodes
 
-As mentioned earlier the Helm chart defaults to expecting at least 2 nodes in the Kubernetes cluster.
+FlowFuse platform expects that Kubernetes nodes have specific labels applied. The main reason behind this approach is to separate core application components from Node-RED instances.
```
Comment (hardillb) [inline docs/install/kubernetes/README.md:142]:
```suggestion
By default FlowFuse platform expects that Kubernetes nodes have specific labels applied. The main reason behind this approach is to separate core application components from Node-RED instances.
```

File: docs/install/kubernetes/README.md [hunk 4]
```diff
@@ -105,132 +112,241 @@ that you set the domain to `example.com`.
...
-```yaml
+```bash
+cat <<EOF > customization.yml
 forge:
   entryPoint: forge.example.com
   domain: example.com
   https: false
   localPostgresql: true
+EOF
 ```
```
Comment (hardillb) [inline docs/install/kubernetes/README.md:139]:
```suggestion
A full list of all the configuration options can be found in the [Helm Chart README](https://github.com/FlowFuse/helm/blob/main/helm/flowforge/README.md#configuration-values).
```

**Review by hardillb — APPROVED** [review]
A couple more suggestions, but looks good.

**Pull request merged**